tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -3434,6 +3428,8 @@ def lwaFirstOffset(self, kernel, tP):
 
     if kernel["LocalWriteUseSgpr%s"%tc]:
       # TODO: Can refactor code above to Compute this directly:
+      if self.states.archCaps["CrosslaneWait"]:
```

Comment:
This is implemented in Tensile, but missing in tensilelite.
It will cause mismatch without s_nop here.